src/scss/_api-symbol.scss

```diff
@@ -6,18 +6,28 @@ $size-medium: 16px;
 $size-large: 24px;
 
 .api-symbol {
-    height: calc($size-medium + 8px);
-    width: calc($size-medium + 8px);
-    font-size: $size-medium;
+    height: calc($size-small + 5px);
+    width: calc($size-small + 5px);
+    font-size: $size-small;
     border-width: 1px;
     border-style: solid;
     box-sizing: border-box;
     border-radius: 50%;
     margin-right: 6px;
-    display: inline-flex;
-    align-items: center;
-    justify-content: center;
+    display: inline-block;
     font-weight: 700;
+    text-decoration: none;
+}
+
+.api-symbol {
+    &:before {
```

Comment:
Just curious -- is there a reason to not place this pseudo-selector inside the existing rule for `.api-symbol` above this?

Reply:
Totally not, just what happens to my brain towards the end of the day.  Just pushed a clean up.